src/util/AttachmentTools.ts

```diff
@@ -0,0 +1,67 @@
+import { Attachment, MeshAttachment, RegionAttachment, TextureAtlasRegion } from "@esotericsoftware/spine-core";
+
+/**
+ * Creates a new `RegionAttachment` from a specified texture atlas region.
+ *
+ * @param region - The texture atlas region used to create the attachment.
+ * @param attachmentName - The name of the new attachment.
+ * @param scale - A scaling factor applied to the attachment's dimensions (default is 1).
+ * @param rotation - The rotation angle of the attachment in degrees (default is 0).
+ * @returns The created `RegionAttachment` with the passed in region.
+ */
+export function createAttachmentFromRegion(
```

Comment:
Use Class to design public API

Reply:
done.
Already put it in AttachmentTools class now.